data_pipelines_cli/data_structures.py

```diff
@@ -81,10 +78,28 @@ def docker_build_tag(self) -> str:
         :return: Tag for Docker Python API build command
         :rtype: str
         """
-        return f"{self.repository}:{self.commit_sha}"
+        return f"{self.repository}:{self.image_tag}"
+
+    def _get_docker_repository_uri_from_k8s_config(self, env: str) -> str:
+        return self._get_docker_image_variable_from_k8s_config("repository", env)
+
+    def _get_image_tag_from_k8s_config(self, env: str, image_tag: Optional[str]) -> str:
+        from data_pipelines_cli.cli_constants import IMAGE_TAG_TO_REPLACE
+
+        if image_tag:
+            return image_tag
```

Comment:
I would replace this with image_tag.getOrElse(self.__get_image_tag_from_k8s_config(env)) What do you think?